test/bin/build_images.sh

```diff
@@ -22,31 +23,31 @@ fi
 
 # Determine the version of the RPM in the local repo so we can use it
 # in the blueprint templates.
-if [ ! -d "${LOCAL_REPO}" ]; then
+if [ ! -d "${SUBST_LOCAL_REPO}" ]; then
     error "Run ${SCRIPTDIR}/create_local_repo.sh before building images."
     exit 1
 fi
-release_info_rpm=$(find "${LOCAL_REPO}" -name 'microshift-release-info-*.rpm' | sort | tail -n 1)
+release_info_rpm=$(find "${SUBST_LOCAL_REPO}" -name 'microshift-release-info-*.rpm' | sort | tail -n 1)
 if [ -z "${release_info_rpm}" ]; then
-    error "Failed to find microshift-release-info RPM in ${LOCAL_REPO}"
+    error "Failed to find microshift-release-info RPM in ${SUBST_LOCAL_REPO}"
     exit 1
 fi
-SOURCE_VERSION=$(rpm -q --queryformat '%{version}' "${release_info_rpm}")
-MINOR_VERSION=$(echo "${SOURCE_VERSION}" | cut -f2 -d.)
-PREVIOUS_MINOR_VERSION=$(( "${MINOR_VERSION}" - 1 ))
-FAKE_NEXT_MINOR_VERSION=$(( "${MINOR_VERSION}" + 1 ))
 
 ## TEMPLATE VARIABLES
 #
-# Machine platform type ("x86_64")
-UNAME_M=$(uname -m)
-export UNAME_M
-export LOCAL_REPO              # defined in common.sh
-export NEXT_REPO               # defined in common.sh
-export SOURCE_VERSION          # defined earlier
-export FAKE_NEXT_MINOR_VERSION # defined earlier
-export MINOR_VERSION           # defined earlier
-export PREVIOUS_MINOR_VERSION  # defined earlier
+SUBST_UNAME_M=$(uname -m) # Machine platform type ("x86_64")
```

Comment:
This feels like it is going to be harder to maintain. Part of the point of using the same variables in the template and in the script is that we don't need to create a bunch of extra variables.

Reply:
OK, I reverted the commit.